react/training-project/src/app/components/Board/index.js

```diff
@@ -1,39 +1,26 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
 import Square from '../Square';
 
-import style from './styles.scss';
-
 class Board extends Component {
-  state = {
-    squares: Array(9).fill(null)
-  };
-  handleClick = i => {
-    const squares = this.state.squares.slice();
-    squares[i] = 'X';
-    this.setState({ squares });
-  };
-
-  renderSquare = i => <Square value={this.state.squares[i]} key={i} id={i} handleClick={this.handleClick} />;
+  renderSquare = i => (
```

Comment:
Why did you add the parenthesis? :thinking:

Reply:
I was trying to make happy the linter :(